actix-web-codegen/src/route.rs

```diff
@@ -0,0 +1,165 @@
+extern crate proc_macro;
+
+use std::fmt;
+
+use proc_macro::TokenStream;
+use quote::{quote};
+
+enum ResourceType {
```

Comment:
i dont think we need this

Reply:
I could use bool, but own enum makes it easier to just impl `Display` for it and use it in formatter

Reply:
Right. It is ok. I misread it